frontend/src/utils/MapCompass.tsx

```diff
@@ -0,0 +1,35 @@
+import { Typography } from '@equinor/eds-core-react'
+import { tokens } from '@equinor/eds-tokens'
+import styled from 'styled-components'
+import { Icons } from 'utils/icons'
+import { Icon } from '@equinor/eds-core-react'
+import { TranslateText } from 'components/Contexts/LanguageContext'
+
+const StyledCompass = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+`
+
+export const MapCompass = (): JSX.Element => {
```

Comment:
Yep, it never moves, I assumed all of the maps are oriented north, if there are other that don't we can create a new issue in the backend/frontend that rotates the map so that north is always up. I think it is more consistent for the users if the maps are always presented with north up

Reply:
Agree, my comment was regarding the name of the component, but not a big problem :)

Reply:
Can change it to NorthArrow if you think that's better :)